example/example.c

```diff
@@ -4,31 +4,24 @@
 
 #include "spglib.h"
 
-static void test_spg_get_symmetry(void);
-static void test_spg_get_symmetry_with_collinear_spin(void);
-static void test_spg_get_multiplicity(void);
-static void test_spg_find_primitive_BCC(void);
-static void test_spg_find_primitive_corundum(void);
-static void test_spg_standardize_cell_BCC(void);
-static void test_spg_standardize_cell_BCC_prim(void);
-static void test_spg_standardize_cell_corundum(void);
-static int sub_spg_standardize_cell(double lattice[3][3], double position[][3],
-                                    int types[], const int num_atom,
-                                    const double symprec,
-                                    const int to_primitive,
-                                    const int no_idealize);
-static void test_spg_get_international(void);
-static void test_spg_get_schoenflies(void);
-static void test_spg_get_spacegroup_type(void);
-static void test_spg_get_magnetic_spacegroup_type(void);
-static void test_spg_get_symmetry_from_database(void);
-static void test_spg_refine_cell_BCC(void);
-static void test_spg_get_dataset(void);
-static void test_layer_spg_get_dataset(void);
-static void test_spg_get_ir_reciprocal_mesh(void);
-static void test_spg_get_stabilized_reciprocal_mesh(void);
-static void test_spg_relocate_BZ_grid_address(void);
-static void test_spg_get_error_message(void);
+static void example_spg_get_symmetry(void);
+static void example_spg_get_symmetry_with_collinear_spin(void);
+static void example_spg_get_multiplicity(void);
+static void example_spg_find_primitive_BCC(void);
+static void example_spg_find_primitive_corundum(void);
+static void example_spg_standardize_cell_BCC(void);
+static void example_spg_standardize_cell_BCC_prim(void);
+static void example_spg_standardize_cell_corundum(void);
+static void example_spg_get_international(void);
+static void example_spg_get_schoenflies(void);
+static void example_spg_get_spacegroup_type(void);
+static void example_spg_get_magnetic_spacegroup_type(void);
+static void example_spg_get_symmetry_from_database(void);
+static void example_spg_refine_cell_BCC(void);
+static void example_spg_get_dataset(void);
+static void example_layer_spg_get_dataset(void);
+static void example_spg_get_ir_reciprocal_mesh(void);
+static void example_spg_get_error_message(void);
```

Comment:
Could you move them to an `example_full.c` and have a single executable example otherwise for this file?

Reply:
Is "a single executable example" considered as an example to build an executable? And `example_full.c` would be examples of how to use API's?

Reply:
Yes, a minimal working example, related to the minimal example that will be in the readme